company_today/data/ir_cron.xml

```diff
@@ -10,9 +10,19 @@ SPDX-License-Identifier: AGPL-3.0-or-later
         <field name="model_id" ref="model_res_company" />
         <field name="code">model.cron_update_today</field>
```

Comment:
please update this branch (merge 12.0 or rebase), to include the fix that actually calls the code.

Reply:
rebased without conflicts